DOCUMENTATION.md

```diff
@@ -99,15 +99,50 @@ This document provides detailed information about each service module and its re
       * [Get Keeper Info](#get-keeper-info)
       * [Get Type And Version](#get-type-and-version-2)
       * [Get Upkeep Transcoder Version](#get-upkeep-transcoder-version)
+  * [Functions Services](#functions-services)
+    * [Service alias: `functionsRouter`](#service-alias-functionsrouter)
+    * [Methods](#methods-8)
+      * [Create subscription](#create-subscription-1)
+      * [Fund subscription](#fund-subscription-1)
+      * [Cancel subscription](#cancel-subscription-1)
+      * [Get subscription details](#get-subscription-details-1)
+      * [Request subscription owner transfer](#request-subscription-owner-transfer-1)
+      * [Accept subscription owner transfer](#accept-subscription-owner-transfer-1)
+      * [Add subscription consumer](#add-subscription-consumer)
+      * [Remove subscription consumer](#remove-subscription-consumer)
+      * [Timeout requests](#timeout-requests)
+        * [Task/Subtask](#tasksubtask)
+        * [HRE](#hre)
+      * [Estimate the cost of a request](#estimate-the-cost-of-a-request)
+      * [Initialize SubscriptionManager class](#initialize-subscriptionmanager-class)
```

Comment:
should initialization (for sub manager + secrets manager) be passed to user? since this plugin is a wrapper and no params are (at the moment) passed to `initialize()` maybe this can just be done under the hood, without imposing this on the user?